packages/calcite-components/src/components/tab-nav/resources.ts

```diff
@@ -0,0 +1,3 @@
+export const CSS = {
+  tabNav: "tab-nav",
```

Comment:
For consistency, can you rename the key to be `container`? We can change the value to match in a follow-up PR.